Tweaks to infra bootstrap (#115)
navapbc/template-infra: infra/README.md

})
```

## Destroying Infrastructure

> To destroy everything you'll need to undo everythin in reverse.

Reviewer: ```suggestion
To destroy everything you'll need to undo everything in reverse.
```

1. First destroy all your environments by running `terraform destroy` in each of your environment module folders

    ```bash
    # within each env folder in /infra/envs/ (dev, stage, etc)
    terraform destroy
    ```

> 2. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the

Reviewer: Should this have more information somewhere on why we need to do this? e.g. indicating that S3 buckets are protected from destruction by default to avoid loss-of-data, and these changes will force the bucket and all contents to be permanently destroyed?

Author: Sounds good, added an explanation

  lifecycle block to set `prevent_destroy = false`. Then run `terraform apply`

    ```terraform

>     resource "aws_s3_bucket" "tf_state" {

Reviewer: should we add a comment to the top of these code blocks indicate what file this is in? it's a little difficult to follow.

      bucket = var.state_bucket_name

      force_destroy = true

      # Prevent accidental destruction a developer executing terraform destory in the wrong directory. Contains terraform state files.
      lifecycle {
        prevent_destroy = false
      }
    }

    ...

    resource "aws_s3_bucket" "tf_log" {
      bucket = var.tf_logging_bucket_name
      force_destroy = true
    }
    ```

> 3. Then since we're going to be destroying the tfstate buckets, you'll want to unconfigure them, so you want to comment out or delete the s3 backend configuration and run `terraform init -force-copy` to copy the tfstate back to a local tfstate file.

Reviewer: ```suggestion
3. Then since we're going to be destroying the tfstate buckets, you'll want to move the tfstate file out of S3 and back to your local system. Comment out or delete the s3 backend configuration and run `terraform init -force-copy` to copy the tfstate back to a local tfstate file.
```

Reviewer: "unconfigure" sounded a little vague to me 👀

Author: sounds good, i was lazily using that wording based on the terraform log output when you do it:

> Terraform has detected you're unconfiguring your previously set "s3" backend.
> Successfully unset the backend "s3". Terraform will now operate locally.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/447859/193302602-4383f8bc-af16-4859-bb77-fad575343488.png">
